packages/client/hmi-client/src/assets/css/theme/designer/components/data/_tree.scss

```diff
@@ -34,6 +36,18 @@
                     }
                 }
 
+                // Added for resource sidebar
+                .p-treenode-label {
+                    text-overflow: ellipsis;
+                    overflow: hidden;
+                    white-space: nowrap;
+                }
```

Comment:
This should not be here only for the resource sidebar.
It's a nice feature to have, but you can remove the comment.